cmd/ceremony/crl_test.go

```diff
@@ -45,14 +46,14 @@ func TestGenerateCRLLength(t *testing.T) {
 	test.AssertEquals(t, err.Error(), "nextUpdate must be less than 12 months after thisUpdate")
 }
 
-type emptySigner struct{}
+type wrappedSigner struct{ k crypto.Signer }
```

Comment:
What purpose does wrapping `k` serve?

Reply:
When we call `Sign` we pass in a `io.Reader` which purposefully fails, because we expect randomness to be coming from the HSM when using PKCS#11 for signing for nonces/RSA blinding. In order to use a real key we need to wrap it so that we can shim in a proper `rand.Reader`, otherwise signing fails.

Reply:
Ah, makes sense! Can you add a comment?